README.md

```diff
@@ -66,11 +66,43 @@ Inspired by the [awesome](https://awesome.re) list. Feel free to improve this li
 
 - [WebdriverIO Community Services](https://github.com/webdriverio-community?q=service) - A set of community maintained services.
```

Comment:
Not sure if this line should remain like this as all the community services are now listed below it.

Reply:
I think it is fine, people will create more services and reporters and might not (or eventually later) add it to this list, so this allows to explore potential plugins not in this list.